.github/workflows/main.yml

```
@@ -1,15 +1,91 @@
   push:
     branches:
       - master
   pull_request:
-    branches_ignore: []
+    branches-ignore: []
 
 jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)